### PR TITLE
Show deal sizes is sealing sectors

### PR DESCRIPTION
**Pull request by magik6k**
2021-09-02-214304_983x126_scrot

(review commit-by-commit)

### CODE REVIEW

File: cmd/lotus-miner/sectors.go [hunk 1]
```diff
@@ -389,112 +413,130 @@ var sectorsListCmd = &cli.Command{
...
+			if !estimate {
+				rdw := big.Add(st.DealWeight, st.VerifiedDealWeight)
+				dw = float64(big.Div(rdw, big.NewInt(int64(st.Expiration-st.Activation))).Uint64())
+				vp = float64(big.Div(big.Mul(st.VerifiedDealWeight, big.NewInt(9)), big.NewInt(int64(st.Expiration-st.Activation))).Uint64())
+			} else {
+				for _, piece := range st.Pieces {
+					if piece.DealInfo != nil {
+						dw += float64(piece.Piece.Size)
+						if piece.DealInfo.DealProposal != nil && piece.DealInfo.DealProposal.VerifiedDeal {
+							vp += float64(piece.Piece.Size) * 9
```
Comment (Stebalien) [inline cmd/lotus-miner/sectors.go:434]: Can we avoid hard-coding 9 here?

File: cmd/lotus-miner/sectors.go [hunk 2]
```diff
@@ -389,112 +413,130 @@ var sectorsListCmd = &cli.Command{
...
+				}
+			}
+
+			if cctx.Bool("events") {
+				var events int
+				for _, sectorLog := range st.Log {
+					if !strings.HasPrefix(sectorLog.Kind, "event") {
+						continue
+					}
+					if sectorLog.Kind == "event;sealing.SectorRestart" {
```
Comment (Stebalien) [inline cmd/lotus-miner/sectors.go:499]: Can we avoid hard-coding?

**Review by Stebalien — APPROVED** [review]
Two nits but this seems like a reasonable refactor. But I haven't _tested_ it so I'll leave that to you.

File: cmd/lotus-miner/sectors.go (hunk 1 above)
Comment (magik6k) [reply cmd/lotus-miner/sectors.go:434]: Somehow that value is really hard to get from actors, though I guess we can at least put this in a constant describing what this is

**Pull request merged**